README.md

```diff
@@ -19,7 +19,7 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://maclenn77.github.io/math-magicians/)
 
 
 ## Getting Started
```

Comment:
- [ ] The link to the screenshot in the Readme file is still broken. Kindly fix it as suggested by the previous reviewer.

Reply:
Addressed this